package.json

```diff
@@ -8,86 +8,82 @@
     "build-schema": "node dist/server/buildSchema.js",
     "test-ci": "npm test",
     "test": "jest \"\\.spec\\.ts\"",
-    "test-integrations": "jest \"\\.integration\\.ts\" --runInBand",
+    "test-integrations": "jest \"\\.integration\\.ts\" --runInBand --forceExit",
```

Comment:
I had to add a forceExit. Theres a handle left open somewhere i could not find :( cc @kschelonka perhaps your wizardry knows where it is??

Reply:
I will take a look!